test/format/backup.c

```diff
@@ -612,8 +622,11 @@ backup(void *arg)
             testutil_check(session->truncate(session, "log:", backup_cursor, NULL, NULL));
 
         testutil_check(backup_cursor->close(backup_cursor));
-        trace_msg(session, "Backup #%u stop%s%s%s", counter, config == NULL ? "" : ": (",
-          config == NULL ? "" : config, config == NULL ? "" : ")");
+        if (config == NULL)
```

Comment:
Definitely an improvement in readability!

Reply:
And it matches the (already existing) "start" trace message. And that's why I made the checkpoint one the same as well. More readable and consistent.